src/test/java/uk/gov/hmcts/sptribs/systemupdate/service/CcdUpdateServiceTest.java

```diff
@@ -13,6 +13,7 @@
 import uk.gov.hmcts.reform.idam.client.models.UserDetails;
 import uk.gov.hmcts.sptribs.ciccase.model.CaseData;
 import uk.gov.hmcts.sptribs.ciccase.model.State;
+import uk.gov.hmcts.sptribs.ciccase.task.CaseTask;
```

Comment:
Can you add some unit test coverage for the exception handling

Reply:
code snippet provided not actually changed in this PR - the additional test will be added by CI task for unit test improvements